owa/app/js/home/home.controller.js

```diff
@@ -35,15 +35,15 @@ class HomeController {
     }
     
     function isFileCorrect(fileName) {
-      return fileName.endsWith(".zip");
+      return fileName.endsWith(".zip") || fileName.endsWith(".json")
     }
 
-    function uploadZip(file) {
+    function uploadZip(file, importType) {
```

Comment:
SHould probably rename this to `uploadFile()`, right?

Reply:
Yes, good catch